spec/features/saml_spec.rb

```diff
@@ -128,14 +123,7 @@
   end
 
   describe 'Users with confirmed CCID accounts' do
-    let!(:user) { FactoryGirl.create :user, email: emailize_uid(OmniAuth.config.mock_auth[:shibboleth][:uid]), ccid: OmniAuth.config.mock_auth[:shibboleth][:uid] }
-    after :each do
-      cleanup_jetty
-    end
-
-    after :all do
-      User.destroy_all
```

Comment:
We are using FactoryGirl....we don't need to clean up all the users? FactoryGirl will roll back the transaction of the user. More unneeded database I/O